test/alternator/suite.yaml

```diff
@@ -1 +1,23 @@
-type: Run
+type: Python
+pool_size: 4
+extra_scylla_cmdline_options:
+#   - '--alternator-address=8080'
+    - '--alternator-enforce-authorization=1'
+    - '--alternator-write-isolation=always_use_lwt'
+    - '--alternator-streams-time-window-s=0'
+    - '--alternator-timeout-in-ms=30000'
+    - '--alternator-ttl-period-in-seconds=0.5'
+#    - '--alternator_port=8000'
+  # Allow testing experimental features. Following issue #9467, we need
+  # to add here specific experimental features as they are introduced.
+  # We only list here Alternator-specific experimental features - CQL
+  # ones are listed in test/cql-pytest/run.py.
+#    - '--experimental-fa   eatures=alternator-streams'
```

Comment:
This shouldn't be commented (and also note the spurious space in the middle). We need to enable alternator streams for the streams tests to run (otherwise, they are all skipped).

Reply:
Default configuration have these parameters that's why we do not need to declare them there.

Reply:
I guess by "default" configuration you refer to the one used today to run CQL. If that's the case, it's weird why it enables the Alternator Streams feature, which is irrelevant to CQL.

Reply:
PythonSuite starts the node and has a default configuration in https://github.com/scylladb/scylladb/blob/master/test/pylib/scylla_cluster.py#L88, so all tests included in PythonSuite have these options enabled.